README.md

```diff
@@ -116,8 +122,15 @@ You can also chat with us via IRC in [#GLPI on freenode](http://webchat.freenode
 Want to file a bug, contribute some code, or improve documentation? Excellent! Read up on our
 guidelines for [contributing](./CONTRIBUTING.md) and then check out one of our issues in the [Issues Dashboard](https://github.com/glpi-project/java-library-glpi/issues).
 
+## Professional Services
+
+The GLPI Network services are available through our [Partner's Network](http://www.teclib-edition.com/en/partners/). We provide special training, bug fixes with editor subscription, contributions for new features, and more.
+
+Obtain a personalized service experience, associated with benefits and opportunities.
+
 ## Copying
 
+* **Name**: GLPI is a registered trademark of Teclib'.
```

Comment:
The hyperlink to the Teclib edition website

Reply:
Link added 👍